### PR TITLE
PMM-9049 Build date

### PR DESCRIPTION
**Pull request by tiagomotasantos**
Clarifies current version and available version dates

https://jira.percona.com/browse/PMM-9049

https://user-images.githubusercontent.com/16366502/147062023-e36d85f6-e678-43a3-8a18-c923d978ad37.mov

### CODE REVIEW

**Comment by rnovikovP** [comment]
adding @rasika-chivate  for an additional 👀  on our text and messages

File: pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.styles.ts [hunk 1]
```diff
@@ -1,32 +1,37 @@
...
-  a {
-    margin-left: 5px;
-  }
-`;
-
-export const availableUpdateVersion = css`
-  font-size: 14px;
-  margin-bottom: 7px;
-`;
+    font-weight: bold;
```
Comment (fabio-silva) [inline pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.styles.ts:8]: should we use `typography.weight.bold` ?

**Review by fabio-silva — APPROVED** [review]

File: pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.styles.ts (hunk 1 above)
Comment (tiagomotasantos) [reply pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.styles.ts]: Done!

**Review by fabio-silva — APPROVED** [review]

**Comment by rnovikovP** [comment]
@tiagomotasantos , what about the case when "no new version available"?
probably we need to add a tooltip to both dates ..as both of the Build dates?

File: pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.test.tsx [hunk 2]
```diff
@@ -1,64 +1,37 @@
...
-      wrapper
-        ?.find('section > p')
-        .at(1)
-        .text(),
-    ).toEqual(`${nextFullVersion} (${nextVersionDate})`);
+    render(<AvailableUpdate nextVersionDetails={nextVersionDetails} />);
+
+    fireEvent.click(screen.getByTestId('update-latest-section'), { altKey: true });
+
+    expect(screen.getByTestId('update-latest-version').textContent).toEqual(nextFullVersion);
```
Comment (atymchuk) [inline pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.test.tsx:35]: Kudos for migrating to the testing-library 👍

**Review by atymchuk — APPROVED** [review]

**Review by rasika-chivate — CHANGES_REQUESTED** [review]
We could rephrase the content a bit:

Releases may not be available on the build date as we run extensive tests (according to the change) in each release. There may be a few days delay between the build and the available dates.

Also,  **Current Version** and **Available Version** is a bit confusing. Instead could we say Current Version and **Next Available Version**.

**Comment by rnovikovP** [comment]
@rasika-chivate, good point, but currently, users are less ok with these terms, and we have plans to rework the Update process and UX for it soon. This is where we'll change wordings. Pls add  your note to https://confluence.percona.com/display/PMM/PMM+Upgrade 

@tiagomotasantos, I see on video 2.26**.0** and 2.27. Where  .0 for   Next Version? Is this just mock, or do we lose it?

**Review by rnovikovP — CHANGES_REQUESTED** [review]
see https://github.com/percona/grafana-dashboards/pull/1294#issuecomment-999514914

**Comment by tiagomotasantos** [comment]
@rnovikovP 

- When no new version is available it will work as before. It was not requested to add tooltip to the Current Version as the tooltip is just to clarify why the build date in the Available Version is a past date (due to the delay between testing and release). If there's a need for a tooltip in Current Version please explain why and what should be added there. I think "Current Version" is clear enough
- Yes the values were just mocks so no ".0" is lost :D 

@rasika-chivate 

Thank you, the tooltip was updated accordingly!
For  Current Version and Available Version we will keep them as they are as @rnovikovP mentioned.

**Comment by tiagomotasantos** [comment]
**UPDATE**

https://user-images.githubusercontent.com/16366502/147331915-52652f31-12db-4ce5-a88b-03c646174d90.mov

**Comment by rnovikovP** [comment]
as for now a lot of text for  dates .. makes it hard to read ( 
Let's warp this to 

```
Current version:   2.26.0 (December 4, 2021)  (?)
Available version: 2.27.0 (December 28, 2021) (?)
```

and place all explanations about dates - in tooltips

**Comment by tiagomotasantos** [comment]
https://user-images.githubusercontent.com/16366502/147350327-139b57d0-f739-4c0e-a934-d5f9d7786d00.mov

**Review by rasika-chivate — CHANGES_REQUESTED** [review]
Rephrase the content on the tooltip as follows:

The date on which the installed version was built

The date on which the installed version was published

**Review by rasika-chivate — DISMISSED** [review]

**Review by denisok — APPROVED** [review]

File: pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.messages.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 export const Messages = {
   availableVersion: 'Available version',
+  tooltip: 'The date on which the version was published',
```
Comment (shoffman-percona) [inline pmm-app/src/pmm-update/components/AvailableUpdate/AvailableUpdate.messages.ts:3]:
```suggestion
  tooltip: 'The date this version was published to the repository',
```

File: pmm-app/src/pmm-update/components/CurrentVersion/CurrentVersion.messages.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export const Messages = {
+  currentVersion: 'Current version',
+  tooltip: 'The date on which the installed version was built',
```
Comment (shoffman-percona) [inline pmm-app/src/pmm-update/components/CurrentVersion/CurrentVersion.messages.ts:3]:
```suggestion
  tooltip: 'The date the currently running version was compiled',
```

**Review by shoffman-percona — COMMENTED** [review]
Just some text clarification (tried not use us built in both the helper text and tooltip)

**Review by denisok — APPROVED** [review]

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**